### DISCUSSION TITLE
Edge Bitwarden Extension Can't Login

### DISCUSSION THREAD

**Discussion by Rvn0xsy** [dani-garcia/vaultwarden#4782] [Q&A]
### Deployment environment

* vaultwarden version: 1.31.0

* Install method: Docker image

* Clients used: web

* Reverse proxy and version:  nginx

* MySQL/MariaDB or PostgreSQL version: sqlite

* Other relevant details:

### Steps to reproduce

I am using the Edge 126.0.2592.113 version of the browser, with the Bitwarden browser extension installed. I tried to log in to my self-hosted server and found that I could not log in normally.
![image](https://github.com/user-attachments/assets/cbd7e50e-f748-4829-a60e-f0bd918da594)

My bitwarden Server version is :

```
root@a4f26485a9b3:/# ./vaultwarden --version
vaultwarden 1.31.0
root@a4f26485a9b3:/#
```

### Expected behaviour

Subsequently, I installed the Chrome browser and also installed the latest Bitwarden browser extension, and found that it works normally!

I would like to inquire whether there are any differences between the Edge browser extension and the Chrome browser extension (the latest version)?

Their browser extension release dates are the same, which is very puzzling to me.

The Chrome browser extension works properly, but the Edge browser does not.

**Comment by BlackDex** [comment]
If you can login via the web-vault and other extensions, then on the Vaultwarden side we can probably not do anything to fix this.
It is either a Browser issue, or the extensions is not working correctly there.

I would suggest to uninstall the extensions, install it again and re-configure it.
Also, check the logs during the login attempt.

**Reply by Rvn0xsy** [reply]
The Bitwarden extension in the Edge add-on store has been updated recently. I tried updating the extension, and it's working properly again :) 

The official update has been made twice within 30 days.
